### PR TITLE
question page added

### PR DESCRIPTION
**Pull request by anand-tiwari**

### CODE REVIEW

**Comment by anand-tiwari** [comment]
Can preview UI here - https://deploy-preview-3--amazing-joliot-58161f.netlify.app/questions/harry-potter.html

**Review by tewarimohit — APPROVED** [review]
Good to merge....

File: common.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    padding: 0px;
+    box-sizing: border-box;
+}
+
+:root {
+    --primary-color: #112a46;
+    --secondary-color: #acc8e5;
+    --shadow-color: #18406d;
+    --white: white;
+    --grey: grey;
```
Comment (SuyashTiwariCoder) [inline common.css:12]: use hex codes for variables .

File: common.css [hunk 2]
```diff
@@ -0,0 +1,52 @@
+* {
+    margin: 0px;
+    padding: 0px;
+    box-sizing: border-box;
+}
```
Comment (SuyashTiwariCoder) [inline common.css:5]: You can use reset.css from your component library

**Pull request merged**